<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_plot</class>
 <widget class="QMainWindow" name="win_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Menu">
   <widget class="QPushButton" name="btnTerm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRadi">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Radix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGrap">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlot">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plotter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTest">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
